### PR TITLE
release note change for update

### PR DESCRIPTION
**Pull request by kalexand-rh**
@ahardin-rh, PTAL?

### CODE REVIEW

File: release_notes/ocp-4-2-release-notes.adoc [hunk 1]
```diff
@@ -71,6 +71,8 @@ See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades
 phased roll out] for more information on the types of updates and upgrade
 channels.
 
+Because upgrades are published to the channels on a time delay as they are gradually rolled out to customers based on data from our SRE team, you might not immediately be able to update from version 4.1.z to 4.2 at initial release.
```
Comment (cuppett) [inline release_notes/ocp-4-2-release-notes.adoc]: They are "able to update", just not via the button consuming suggestions from the cloud.  We might alter the second half to say "you might not immediately see notification in the UI at initial release" to more accurately capture what's going on there.

File: release_notes/ocp-4-2-release-notes.adoc [hunk 2]
```diff
@@ -71,6 +71,8 @@ See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades
...
+
```
Comment (ahardin-rh) [inline release_notes/ocp-4-2-release-notes.adoc]: Red Hat Service Reliability Engineering (SRE)

**Comment by ahardin-rh** [comment]
One comment from me. Otherwise, looks great!

File: release_notes/ocp-4-2-release-notes.adoc (hunk 1 above)
Comment (mbearer) [reply release_notes/ocp-4-2-release-notes.adoc]: Though it still begs the question of when the fast channel upgrade will come, here are some thoughts: 

Red Hat recommends [or We recommend] timing your upgrade from 4.1.z to 4.2.0 to coincide with a fast channel upgrade. Fast channel upgrades are published on a time delay and gradually rolled out to customers based on data from our SRE team, which is why you might not see notification in the UI at the initial release time.

Alternately you could add a sentence warning them that if they decide to proceed with an upgrade, we strongly recommend doing a full backup?

File: release_notes/ocp-4-2-release-notes.adoc (hunk 2 above)
Comment (cuppett) [reply release_notes/ocp-4-2-release-notes.adoc]: I would lose "on a time delay" and just say "upgrades are gradually rolled out to customers". It's not a "time delay" at all, it's based on the data, period.

Comment (cuppett) [reply release_notes/ocp-4-2-release-notes.adoc]: (didn't see the updated commit, these current words are +1 from me)

Comment (sferich888) [reply release_notes/ocp-4-2-release-notes.adoc]: +1

Comment (vikram-redhat) [reply release_notes/ocp-4-2-release-notes.adoc]: Need acks from @reestr, and possibly someone from the BU. Possibly @mpbarrett or @knewcomerRH .

I believe eng has given the ack via @cuppett (here) and @calfonso (via email).

**Review by reestr — COMMENTED** [review]
LGTM

File: release_notes/ocp-4-2-release-notes.adoc (hunk 2 above)
Comment (reestr) [reply release_notes/ocp-4-2-release-notes.adoc]: LGTM

**Comment by vikram-redhat** [comment]
Got ack from the majority of stakeholders. Merging.

**Pull request merged**